auth/nginx.conf

```
@@ -0,0 +1,65 @@
+#Template from https://docs.docker.com/registry/recipes/nginx/
+
+events {
+    worker_connections  1024;
+}
+
+http {
+
+  upstream docker-registry {
+    server registry:5000;
+  }
+
+  ## Set a variable to help us decide if we need to add the
+  ## 'Docker-Distribution-Api-Version' header.
+  ## The registry always sets this header.
+  ## In the case of nginx performing auth, the header is unset
+  ## since nginx is auth-ing before proxying.
+  map $upstream_http_docker_distribution_api_version $docker_distribution_api_version {
+    '' 'registry/2.0';
+  }
+
+  server {
+    listen 443 ssl;
+    server_name docker.cdot.systems;
```

I think we should use a variable here, in case we decide to move it somewhere else.
```suggestion
    server_name ${DOMAIN};
```